app/controllers/articles_controller.rb

```diff
@@ -241,8 +241,9 @@ def article_params_json
 
     # handle series/collections
     if params["article"]["series"].present?
-      params["article"]["collection_id"] = Collection.find_series(params["article"]["series"], @user)&.id
-    elsif params["article"]["series"] == ""
+      collection = Collection.find_series(params["article"]["series"], @user)
```

Comment:
So, the collection is the same thing as the series? :thinking:

Reply:
yeah, we're using two domain names for the same thing

Reply:
My longterm thought was that a "series" could be a type of collection. E.G. there could be other reasons to bundle articles. Never wound up implementing the other stuff, but who knows maybe someday.